tests/mocha.opts

```diff
@@ -1,2 +1,3 @@
 --colors
+--compilers js:pouchdb-plugin-helper/mocha-babel 
```

Comment:
The tests were originally written in Python. I ported them to ES6 since it allowed for the most direct translation. They depend quite heavily on async/await, which as far as I know is not supported even in node 4+.